sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -90,7 +121,7 @@ public virtual Response Delete(CancellationToken cancellationToken = default)
 
             try
             {
-                return Operations.StartDelete(Id.Name, cancellationToken).WaitForCompletion(cancellationToken);
+                return RestClient.Delete(Id.Name, cancellationToken);
```

Comment:
the delete method should call the StartDelete method then WaitForCompletion

Reply:
The RestClient changed to the type of ResourceGroupsRestOperatuins now, there's no lro for delete in that class.